drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -201,6 +229,8 @@ def process(transaction):
         currentPath = os.path.realpath(os.path.join(incomingPath,f))
         if os.path.isdir(currentPath):
             nanoporeFolder = currentPath
+        if currentPath.endswith('.sha256sum'):
+            fillChecksumMap(currentPath)
```

Comment:
I do not understand this method.
Are you parsing a file for `tokens`(what are those?) or are you providing a list of file paths and file paths to their checksums?

Reply:
for each incoming folder/file (= transaction) a file ending in .sha256sum is created that contains checksums for each file that is part of this transaction along with their relative paths. these are filled into the map. I think commenting on that further up should clarify it?